gateway-test-release-utils/src/main/java/org/apache/knox/gateway/GatewayTestConfig.java

```diff
@@ -91,6 +91,11 @@ public String getGatewaySecurityDir() {
     return gatewayHomeDir + "/security";
   }
 
+  @Override
+  public String getGatewayKeystoreDir() {
+    return getGatewaySecurityDir() + "/keystores";
```

Comment:
Use Paths.get or non os specific separator

Reply:
I thought about that since I did it other places, but it goes against the general style of the file... so I didn't do it that way.  I will fix though.

Reply:
Should I convert all of the path concatenations in the file?

Reply:
Ew - didn't realize that there were other examples that are hardcoded with `/`. I think either we fix them all in this file or don't fix it with this PR and do it in a followup.

Reply:
I am happy to fix them all now.